src/main/java/crazypants/enderio/BlockEio.java

```diff
@@ -95,6 +95,10 @@ public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer e
       }
     }
     
+    if (entityPlayer.isSneaking()) {
```

Comment:
I did not want to write

    if (!entityPlayer.isSneaking()) {
        code(1)
    }
    if (entityPlayer.isSneaking()) {
        code(2);
    }
    code3();

and I didn't want to put the remainder of the method into an "else" bacause that would change indent-level and I know, you don't like that in commits.